src/components/Main/MainContent.tsx

```diff
@@ -0,0 +1,38 @@
+import "./main.scss";
+import React from "react";
+import { useSelector } from "react-redux";
+import { RootStore } from "../../redux/store";
+import { JobType } from "../../redux/jobs/jobs-action-types";
+
+const MainContent: React.FC = () => {
+  const jobState = useSelector((state: RootStore) => state.job);
+  console.log(jobState);
+  return (
+    <>
+      <h2>Top Jobs</h2>
+      <div className="d-flex flex-wrap justify-content-around">
+        {jobState.job?.map((item: JobType) => {
+          return (
+            <div key={item.id} className="card">
+              <img
+                width="170"
+                height="170"
```

Comment:
Is this work? Look like it is incorrect

Reply:
yes this work, i've try to remove this and then 
![image](https://user-images.githubusercontent.com/62364608/160049986-28e54610-82b0-4efd-8827-eac60e74ec7d.png)